### DISCUSSION TITLE
PAGViewer工具中文字体样式设置不生效

### DISCUSSION THREAD

**Discussion by zbin95** [Tencent/libpag#1646] [Q&A] [answered]
我正在使用c++代码做一些开发，在替换字体时遇到中文字体替换不生效的问题，例如【楷体】，另外我发现在PAGViewer工具中文字体样式设置后同样不生效，我目前使用的都是系统字体，请问是c++的相关代码还不支持中文字体的设置么

**Comment by kevingpqi123** [comment, marked as answer]
如果需要显示特定字体，需要字体注册，可以看下这篇文档：https://pag.art/docs/api-instructions.html#pag-%E5%AD%97%E4%BD%93%E6%B3%A8%E5%86%8C

**Reply by zbin95** [reply]
好的，谢谢，不过我要使用的是系统字体，经过测试我发现应用【楷体】时C++代码需要传入字体的英文名称【KaiTi】，应该可以解决我的问题
